src/main/java/com/tekenable/config/AuditConfigurator.java

```diff
@@ -10,26 +14,24 @@
 
 public class AuditConfigurator {
```

Comment:
That's fine and it's actually a good idea except the commit stuff which is not doable. 
I can imagine some kind of "rollback" method doing the clean-up in case of any failure but that's going to be a pain.
When anything goes wrong here it is better to allow stopping the build and reverse everything as it is now.